README.md

```diff
@@ -1,7 +1,10 @@
 # preact-router
 
 [![NPM](https://img.shields.io/npm/v/preact-router.svg)](https://www.npmjs.com/package/preact-router)
-[![travis-ci](https://travis-ci.org/developit/preact-router.svg)](https://travis-ci.org/developit/preact-router)
+[![Build status][github-ci-badge]][github-ci-url]
+
+[github-ci-badge]: https://github.com/preactjs/preact-router/actions/workflows/node.js.yml/badge.svg
+[github-ci-url]: https://github.com/preactjs/preact-router/actions/workflows/node.js.yml
```

Comment:
```suggestion
[![Build status](https://github.com/preactjs/preact-router/actions/workflows/node.js.yml/badge.svg)](https://github.com/preactjs/preact-router/actions/workflows/node.js.yml)
```

Same thing as on `render-to-string`, extracting the link out only creates inconsistency in the document and isn't really needed as neither are referenced more than once.

Reply:
Updated as suggested.